sdformat_mjcf/README.md

```diff
@@ -59,3 +58,16 @@ extract the contents and run
 ```
 <path/to/mujoco>/bin/simulate new_file.xml
 ```
+
+To convert an SDFormat file to mjcf:
+
+```
+mjcf2sdformat path/to/file.xml new_file.sdf
+```
+
+To run the sdf file in GazeboSim, follow [this instructions to install Gazebo Sim](https://gazebosim.org/docs/latest/install)
```

Comment:
```suggestion
To run the sdf file in GazeboSim, follow [these instructions to install Gazebo Sim](https://gazebosim.org/docs/latest/install)
```

Reply:
Done in https://github.com/gazebosim/gz-mujoco/pull/74/commits/c220e91d2e5279579e07af08b558d15347f891a5